src/client/camera.cpp

```diff
@@ -628,7 +628,7 @@ void Camera::wield(const ItemStack &item)
 void Camera::drawWieldedTool(irr::core::matrix4* translation)
 {
 	// Clear Z buffer so that the wielded tool stays in front of world geometry
-	m_wieldmgr->getVideoDriver()->clearBuffers(video::ECBF_DEPTH);
+	// m_wieldmgr->getVideoDriver()->clearBuffers(video::ECBF_DEPTH);
```

Comment:
if this is not needed anymore please remove it whole

Reply:
Removed.